accern_xyme/accern_xyme.py

```diff
@@ -826,48 +840,89 @@ def dynamic_async_obj(
         ])
 
     def get_dynamic_result(self, data_id: str) -> ByteResponse:
-        cur_res, ctype = self._client.request_bytes(
-            METHOD_GET, "/dynamic_result", {
-                "pipeline": self._pipe_id,
-                "id": data_id,
-            })
+        from requests.exceptions import HTTPError
+        try:
+            cur_res, ctype = self._client.request_bytes(
+                METHOD_GET, "/dynamic_result", {
+                    "pipeline": self._pipe_id,
+                    "id": data_id,
+                })
+        except HTTPError as e:
```

Comment:
handle Server error, no?

Reply:
ServerSideError? that's in interpret_ctype following